app/models/qdm/basetypes/data_element.rb

```diff
@@ -18,9 +18,9 @@ class DataElement
 
     def initialize(options = {})
       # class is reserved word. changed to clazz
-      if options[:class]
-        options[:clazz] = options[:class]
-        options.delete(:class)
+      if options['class']
```

Comment:
I think it was _class somewhere else.. Or maybe that was in FHIR

Reply:
I think HAPI FHIR has class_ 
in bonnie fhir models, we have class_local

Reply:
right..
np. not relevant.